core/src/solve.jl

```diff
@@ -478,34 +479,76 @@ function continuous_control!(
 
         listened_node_id = listen_node_id[i]
         _, listened_node_idx = id_index(basin.node_id, listened_node_id)
+        storage_listened_basin = u.storage[listened_node_idx]
+        phi = storage_listened_basin < 10.0 ? storage_listened_basin / 10.0 : 1.0
 
         flow_rate = 0.0
 
         if !isnan(proportional[i])
-            flow_rate += proportional[i] * error[i]
+            flow_rate += phi * proportional[i] * error[i]
+        end
+
+        if !isnan(integral[i])
+            # coefficient * current value of integral
+            flow_rate += phi * integral[i] * integral_value[i]
         end
 
         if !isnan(derivative[i])
             # dlevel/dstorage = 1/area
             area = basin.current_area[listened_node_idx]
-
-            error_deriv = -dstorage[listened_node_idx] / area
-            flow_rate += derivative[i] * error_deriv
-        end
-
-        if !isnan(integral[i])
-            # coefficient * current value of integral
-            flow_rate += integral[i] * integral_value[i]
+            dlevel = du.storage[listened_node_idx] / area
```

Comment:
Not used?

Reply:
Well that's interesting....

Reply:
Ah `du.storage[listened_node_idx]` is used lateron